jboss-kie-common/configure.sh

```diff
@@ -12,3 +12,8 @@ cp -r ${ADDED_DIR}/launch/* ${JBOSS_HOME}/bin/launch
  # Set bin permissions
 chown -R jboss:root ${JBOSS_HOME}/bin/
 chmod -R g+rwX ${JBOSS_HOME}/bin/
+
+##we create an empty known_hosts global if the user forgot to pass the known-hosts with the id_rsa
+mkdir /etc/ssh/
+>/etc/ssh/known_hosts
```

Comment:
reading more about the purpose of this file I am not sure if it make sense. 
does the business central uses this file instead the one on user's home? And, how this file will contains the users information about remote git repos?

Reply:
Basically we are showing a msg to the user to communicate that a file was missing when the secret was create and we are providing a empty file, the current behaviour is stuck on the start and no msgs, only in the events the root cause could be readed

Reply:
@spolti Honestly I don't remember this very well, LOL, but IIRC, yeah I think it is assuming that location.

Reply:
what this empty file will change?

Reply:
Again, I don't remember very well, but I *thought* that the user has the ability to use a config map to overlay on that location to specify their own file content. But in the case they *don't* do that, things would fail if the file didn't exist.

Reply:
I could be wrong, though... I'd have to take some time to dig into it more.

Reply:
@desmax74 is the issue gone or it just masks the problem that user will have later when trying to use githooks?
if it just allow bc to starts if the know_hosts is not there, maybe we could rely on ocp events to let user know about it. It can be done on operator side.

 (i suppose that it is mandatory for github or other git services to provide the exchange key and save it on the know hosts, right?)

Reply:
@spolti The issue can't gone until the user provides its correct known_host or it add the right content in the /etc/ssh/known_host (with right permissions to edit), here we have removed the known_hosts to permit the startup of the pod https://github.com/kiegroup/kie-cloud-operator/pull/524 after discussion with @tchughesiv  (note, the /home/jboss/.ssh can't be used to add known_hosts by us because is the volume mounted for the secrets) and these changes are less intrusive to change code on operator.

Reply:
if user must provide it, let's only print a message on the logs **and** add a message in the ocp events board.
lets drop the latest changes.

Reply:
this way user will have two places to check the status, events would be better though,
@tchughesiv

Reply:
@spolti if there's no downside to providing an empty known_hosts file, this would appear to be the best route. it also allows the user to define their own known_hosts should they want to. the issue here is that we don't create this particular secret in the operator code. the user does this as a manual step so we can't really enforce it with operator.